src/webots/control/WbLanguageTools.hpp

```diff
@@ -35,6 +35,9 @@ class WbLanguageTools {
 private:
   WbLanguageTools() {}
   ~WbLanguageTools() {}
+  static const QString checkIfPythonCommandExist(const QString &pythonCommand, QProcessEnvironment &env, bool log);
+  // cppcheck-suppress unusedPrivateFunction
```

Comment:
Instead of `cppcheck-suppress unusedPrivateFunction`, we should declare and implement this function only in the macOS case (using `#ifdef`).